### ISSUE TITLE
Drillup "Cannot set property 'length' of undefined"

### ISSUE THREAD

**Issue by balkarov** [blacklabel/custom_events#114]
When I add click event to `plotOptions.series.point` and call `chart.drillUp()` I get error

> Drillup "Cannot set property 'length' of undefined".
```
plotOptions: {
        series: {
            point: {
                events: {
                    click: function () {

                    },
               }
          }
    }
}
```

Without `plotOptions.series.point.click` there is no error, but I need the click event

**If do not use custom_events then all ok.
Look [example ](https://jsfiddle.net/qk7b9do1/)**

#### Expected behaviour
Without error

#### Actual behaviour
Has error 
![error](https://user-images.githubusercontent.com/6030335/36972361-da148912-207f-11e8-9e08-cc5faa11e86c.png)

#### Live demo with steps to reproduce
https://jsfiddle.net/0t21eyha/

#### Product version
Highcharts custom events version 2.1
Highcharts version 6.0.7

#### Affected browser(s)
Chrome 64.0.3282.167

**Comment by sebastianbochan** [comment]
Hi @balkarov 
Thank you for the reporting, I will take care about it as soon as possible.

**Comment by sebastianbochan** [comment]
Hi @balkarov 
With the newest custom-events I was not able to replicate the issue. Could you confirm that problem still exist?

Chrome  64.0.3282.186

**Comment by balkarov** [comment]
Hello. 
Yes, it does. 
You can check live demo.

**Comment by white91wolf** [comment]
i just had an exception in a pie chart where elemObj is undefined and in line 344 there is no check if elemObj is defined and a method gets called -> so there is an exception. i fixed it quickly by adding a defined check like the if's above.

if (elemObj && elemObj.drilldown) {

**Comment by sebastianbochan** [comment]
Hi @white91wolf,
Thank you for feedback. Did you notice something unexpected when you fix that in this way?
Could you attach a demo of unworking scenario?

 I will fix this after a feedback.

**Comment by white91wolf** [comment]
sorry for the late answer. i have to say i dont use the drilldown or something like that just normal highcharts. on my page i have 8 charts and on column charts i added a context menü - but i also had to add for every chart the click events to close/hide the context menu and on the click on the pie chart i got the exception. 
i also use the Highcharts Version 4.1.3. I just commented here cause of the comment in the customEvents.js ref to this issue. i think the simple null check should not be a problem or cause any side effects.

**Comment by sebastianbochan** [comment]
@white91wolf 
Thank you for feedback. I added this small fix and publish new `v2.2.1` version. If problem will come again, please simplify chart as live demo and reopen the case.
